package.json

```diff
@@ -6,7 +6,7 @@
     "json"
   ],
   "homepage": "https://github.com/Leonidas-from-XIV/node-xml2js",
-  "version": "0.4.13",
+  "version": "0.4.14",
```

Comment:
Could you not bump the version? I was asked by people using the git repo directly to only bump it directly before the release. But you could add yourself to the contributors!